ResearchKit/Common/ORKNavigableOrderedTask.m

```diff
@@ -206,7 +206,7 @@ - (ORKStep *)stepBeforeStep:(ORKStep *)step withResult:(ORKTaskResult *)result {
 
     if (indexOfCurrentStepResult != -1 && indexOfCurrentStepResult != 0 ) {
```

Comment:
friendly comment: try to avoid magic numbers :)

Reply:
gotcha ;)